stream-chat-android-ui-components/build.gradle

```diff
@@ -83,7 +83,6 @@ dependencies {
     implementation Dependencies.androidLegacySupport
     implementation Dependencies.constraintLayout
     implementation Dependencies.materialComponents
-    implementation Dependencies.drawabletoolbox
```

Comment:
This was simply unused

Reply:
Should we also remove it from `Dependencies`?

Reply:
Used by the old UI, so we can't get rid of it quite yet. I'm not sure if I want to replace it with Material stuff there, or just leave the old implementation be and not break it.